src/io/flutter/inspector/DiagnosticsNode.java

```diff
@@ -28,11 +27,7 @@
 import javax.swing.*;
 import java.util.*;
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import static io.flutter.sdk.FlutterSettingsConfigurable.WIDGET_FILTERING_ENABLED;
```

Comment:
Filtering in IntelliJ removed as filtering is now performed in package:flutter where it is more efficient.